src/ui/MLFacet/MLFacet.ts

```diff
@@ -248,6 +247,28 @@ export class MLFacet extends Component {
     this.handleFacetValuesChanged();
   }
 
+  /**
+   * Requests an additional amount of values equal to the [`number of values`]{@link MLFacet.options.numberOfValues}.
+   *
+   * Does trigger a query automatically.
+   */
+  public showMoreValues(): void {
+    this.ensureDom();
+    this.logger.info('Show more values', (this.mLFacetQueryController.numberOfValuesToRequest += this.options.numberOfValues));
```

Comment:
I think it would make sense to add methods on the controller class `increaseNumberOfValuesToRequest()`, `decreaseNumberOfValuesToRequest` to keep the logic more encapsulated.

Reply:
Nice and clean!